packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -7,13 +7,21 @@ import { get } from 'lodash';
  * WordPress dependencies
  */
 import {
-	__experimentalNavigation as Navigation,
-	__experimentalNavigationMenu as NavigationMenu,
-	__experimentalNavigationItem as NavigationItem,
+	__experimentalNavigator as Navigator,
```

Comment:
Just flagging that, with #35160 merged, this PR should be rebased and `Navigator` should be renamed to `NavigatorProvider`